srcpkgs/setxkbmap/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'setxkbmap'
 pkgname=setxkbmap
-version=1.3.2
+version=1.3.3
 revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config"
-makedepends="libX11-devel libxkbfile-devel"
+makedepends="libX11-devel libxkbfile-devel libXrandr-devel"
 short_desc="Set the keyboard map using the X Keyboard Extension"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="http://xorg.freedesktop.org"
```

Comment:
development moved to https://gitlab.freedesktop.org/xorg/app/setxkbmap , can you change homepage?